frontend/src/scenes/project/Settings/index.tsx

```diff
@@ -16,18 +16,51 @@ import { CodeSnippet } from 'scenes/ingestion/frameworks/CodeSnippet'
 import { teamLogic } from 'scenes/teamLogic'
 import { DangerZone } from './DangerZone'
 import { PageHeader } from 'lib/components/PageHeader'
+import { Link } from 'lib/components/Link'
+import { commandPaletteLogic } from 'lib/components/CommandPalette/commandPaletteLogic'
+import { CommentOutlined, SendOutlined, CheckOutlined } from '@ant-design/icons'
+import { userLogic } from 'scenes/userLogic'
+import posthog from 'posthog-js'
 
 export const Setup = hot(_Setup)
-function _Setup({ user }) {
+function _Setup(): JSX.Element {
     const { currentTeam } = useValues(teamLogic)
     const { resetToken } = useActions(teamLogic)
     const { location } = useValues(router)
+    const { user } = useValues(userLogic)
+
+    const { activateFlow, showPalette } = useActions(commandPaletteLogic)
+
+    const shareFeedback = (instruction: string = "What's on your mind?"): void => {
+        showPalette()
+        activateFlow({
```

Comment:
Aha - didn't see this when originally reviewing.

This shouldn't live in this component, but as an action/listener under commandPaletteLogic.

Reply:
Addressed on #2639

Reply:
TBH I'd prefer this to live inside the component. Overall with the current palette logic approach everything goes into `commandPaletteLogic`, which gets more and more bloated with every release.